packages/babylon/src/plugins/typescript.js

```diff
@@ -64,6 +64,8 @@ function keywordTypeFromName(
 
 export default (superClass: Class<Parser>): Class<Parser> =>
   class extends superClass {
+    +jsxTagStartToRelational: () => void;
```

Comment:
Sorry for my ignorance but what is this syntax?

Reply:
It means that that method is defined in another class (a subclass or a superClass that Flow can't infer) but it will available at runtime.

Reply:
@xtuc I just read the docs and I was wrong :sweat_smile:  It is just a read-only type annotation, which can be used to declare methods which flow can't infer.

Reply:
ok :smile: thanks